libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -631,6 +639,17 @@ void RangeFinder::handle_msg(const mavlink_message_t &msg)
     }
 }
 
+#if AP_SCRIPTING_ENABLED
+void RangeFinder::handle_script_msg(float dist_m)
```

Comment:
You could add a singleton for the scripting backend and go direct. But in that case you can only have one scripting backend. I guess the goal of this would be to make it possible to have multiple backends?

Reply:
@IamPete1 that's what I was thinking. Maybe someone wants to have multiple lua scripting rangefinder drivers..

Reply:
Infact, since you can now get the backend pointer directly you do not need to go via the front end at all.